vowpalwabbit/parser.cc

```diff
@@ -698,6 +698,12 @@ void setup_example(vw& all, example* ae)
   // If this example has a test only label then it is true regardless.
   ae->test_only |= all.example_parser->lbl_parser.test_label(&ae->l);
 
+  if(all.privacy_activation)
+  {
+    ae->tag_hash=hashall(ae->tag.begin(),ae->tag.size(),all.hash_seed)%32;
```

Comment:
for `32` I'd add a variable next to `privacy_activation` and hardcoded it there to `32` so that when we make it configurable we just need to set it there via cli